src/routes/blog/post/go-function-benchmarks/+page.markdoc

```diff
@@ -0,0 +1,225 @@
+---
+layout: post
+title: Measuring Appwrite's Go runtime performance
+description: How we benchmarked the performance of our new Go runtime in comparison with other Appwrite Functions runtimes
+date: 2024-10-08
+cover: /images/blog/go-function-benchmarks/cover.png
+timeToRead: 8
+author: matej-baco
+category: open-source
+---
+
+It is undeniable that Go has grown to become one of the most popular programming languages among developers worldwide. Recently, during Init, we announced the introduction of the new Golang (or Go) runtime for Appwrite Functions. However, it is one thing for us to claim that our Go functions runtime is performant, it is a whole other thing for us to justify the same. To do so, we planned a benchmark to test the performance of our Go runtime in comparison with other Appwrite Functions runtimes.
```

Comment:
let's link to the announcement blog: https://appwrite.io/blog/post/announcing-go-support